clientv3/balancer/balancer_test.go

```diff
@@ -277,6 +285,18 @@ func TestRoundRobinBalancedResolvableFailoverFromRequestFail(t *testing.T) {
 		return picked, err
 	}
 
+	var picked string
+	available := make(map[string]struct{})
+	// cycle through all peers to indirectly verify that balancer subconn list is fully loaded
+	// otherwise we can't reliably count switches in the next step
+	for len(available) < serverCount {
+		picked, err = reqFunc(context.Background())
+		if err != nil {
+			t.Fatalf("Unexpected failure %v", err)
+		}
+		available[picked] = struct{}{}
+	}
```

Comment:
I see exact the same code multiple times, can you get them wrapped in a common function?

Reply:
I've followed the same style as original tests, there is a bunch of duplication.